script/common/Vat.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.21;
+
+import { Ownable } from "openzeppelin-contracts/contracts/access/Ownable.sol";
+
+contract Vat is Ownable {
+
+    uint256 public ilkLine;
+
+    constructor(address _owner) Ownable(_owner) {
+        ilkLine = 1e9 * 1e45;  // Just make it some really large number by default so we can ignore
+    }
+
+    function ilks(bytes32) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust) {
+        return (0, 0, 0, ilkLine, 0);
```

Comment:
default rate should be `10 ** 27` rather than 0. [Vat](https://github.com/makerdao/dss/blob/master/src/vat.sol#L102). For now we implemented a workaround in the planner.

Reply:
fixed